### ISSUE TITLE
Bossbar bugg

### ISSUE THREAD

**Issue by lukrr** [kangarko/Boss#1354]
Hello, I updated the plugin from version 5.1.1 to 5.2.4 and found a bug. When hitting the boss, the name in the boss bar doesn’t appear with color; instead, it appears in white. I use the "BossBarHealth" plugin to display the bossbar of mobs. When I was using version 5.1.1, the name appeared correctly with color, but after updating this plugin, it stopped working.
(Regardless of whether I use hex colors or vanilla colors, in both cases, it appears in white)

![Screenshot_1](https://github.com/user-attachments/assets/ef9bbb35-a650-4be6-86d9-1685fe67b13f)

**Comment by lukrr** [comment]
5.1.1 works fine
![Screenshot_1](https://github.com/user-attachments/assets/eecdda9a-945a-4934-9bf3-35bd0d100541)

**Comment by kangarko** [comment]
Yes that was an intentional change as requested by other users. I can add one more placeholder with color, can you tell me which placeholder are you using for the boss name?

**Comment by lukrr** [comment]
In bossbar plugin? this
![imagen](https://github.com/user-attachments/assets/99b1bb6d-49ce-4248-9ac5-26f1ec634e7c)

**Comment by kangarko** [comment]
Please allow 48 hours for investigation

**Comment by lukrr** [comment]
Ok thanks (:

**Comment by kangarko** [comment]
Quick question is this a Player type of Boss and you're using Citizens?

**Comment by lukrr** [comment]
I don't have citizens installed

**Comment by kangarko** [comment]
Unfortunately I found it technically impossible to distinguish between a colorized and non colorized display name, the change will remain as is because other people preferred it and I have to cater to majority votes

**Comment by lukrr** [comment]
So there is no way I can display the boss color on the bossbar?
